Integrations/SecBI/SecBI.py

```diff
@@ -0,0 +1,193 @@
+        """
+        Get a specific SecBI Incident by Host
+        :param host: The host by which to get a SecBI Incident
+        :return:  A dictionary representation of the SecBI Incident found to contain the supplied host
+        """
+        params = assign_params(host=host, fields=SECBI_INCIDENT_FIELDS)
+        return self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_INCIDENT, params=params)
+
+    def test_module(self):
+        """
+        Performs basic get request to test SecBI system
+        """
+        return self._http_request(method='GET', url_suffix=SECBI_API_ENDPOINT_STATUS)
+
+
+def test_module_command(client: SecBIClient):
+    """
+    This is the call made when pressing the integration test button.
+    Performs basic get request to test SecBI system
+    :param client: The SecBI client to use for the sanity test
+    """
+    results = client.test_module()
+    if 'status' in results and results['status'] == 'ACTIVE':
+        demisto.results('ok')
+    raise DemistoException("Failed connection test of SecBI. Please check your SECBI_API_URL and SECBI_API_KEY")
+
+
+def secbi_get_incidents_list_command(client: SecBIClient, args: Dict) -> Tuple[str, Dict, Dict]:
+    """
+    SecBI Get All Incident IDs command
+    :param client: The SecBI client to use
+    :param args: The Demisto args
+    :return: Content for return_outputs()
+    """
+    query = args.get('query', None)
+    limit = args.get('limit', 100)
+    incidents_list = client.secbi_get_incidents_list(query, limit)
+
+    formatted = [dict(ID=s) for s in incidents_list]
+    human_readable = tableToMarkdown('List of SecBI Incidents', formatted)
+    entry_context = {
+        'SecBI.IncidentsList': incidents_list
+    }
+    return human_readable, entry_context, incidents_list
+
+
+def secbi_get_incident_command(client: SecBIClient, args: Dict) -> Tuple[str, Dict, Dict]:
+    """
+    SecBI Get Incident command
+    :param client: The SecBI client to use
+    :param args: The Demisto args
+    :return: Content for return_outputs()
+    """
+    incident_id = args.get('incident_id', None)
+    incident_data = as_secbi_incident(client.secbi_get_incident(incident_id))
+
+    human_readable = tableToMarkdown('SecBI incident ID "{}"'.format(incident_id), incident_data)
+    entry_context = {
+        'SecBI.Incident(val.ID === obj.ID)': incident_data
+    }
+    return human_readable, entry_context, incident_data
+
+
+def secbi_get_incident_by_host_command(client: SecBIClient, args: Dict) -> Tuple[str, Dict, Dict]:
+    """
+    SecBI Get Incident by Host command
+    :param client: The SecBI client to use
+    :param args: The Demisto args
+    :return: Content for return_outputs()
+    """
+    host = args.get('host', None)
+    incident_data = as_secbi_incident(client.secbi_get_incident_by_host(host))
+
+    human_readable = tableToMarkdown('SecBI incident by host "{}"'.format(host), incident_data)
+    entry_context = {
```

Comment:
Please also populate output according to [IP context standard](https://github.com/demisto/content/tree/master/docs/context_standards#ip) and [Host context standard](url)

Reply:
These were not currently planned use-cases by our product team at this time.